pkg/minikube/openshiftversions/openshift_versions.go

```diff
@@ -36,7 +36,7 @@ func PrintOpenShiftVersionsFromGitHub(output io.Writer) {
 func PrintOpenShiftVersions(output io.Writer) {
 	versions, err := getVersions()
 	if err != nil {
-		glog.Errorln(err)
```

Comment:
@hferentschik Agreed, we might have to look https://github.com/docker/machine/tree/master/libmachine/log